scripts/mirror-actors/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "mirror-actors"
+version = "0.1.0"
+description = ""
+authors = ["samuelarogbonlo <samuel@chainsafe.io>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.12"
+requests = "^2.31.0"
+slack-sdk = "^3.26.2"
+boto3 = "^1.34.22"
```

Comment:
There's also `boto3-types[S3]` iirc which helps pylance. Might be worth adding as a dev dep